runelite-client/src/main/java/net/runelite/client/plugins/world/WorldPlugin.java

```diff
@@ -36,46 +37,57 @@
 import net.runelite.client.config.ConfigManager;
 import net.runelite.client.plugins.Plugin;
 import net.runelite.client.plugins.PluginDescriptor;
+import net.runelite.client.ui.overlay.OverlayManager;
 import net.runelite.client.util.WorldUtil;
 import net.runelite.http.api.worlds.World;
 import net.runelite.http.api.worlds.WorldClient;
 import net.runelite.http.api.worlds.WorldResult;
 
 @PluginDescriptor(
-	name = "Default World",
-	description = "Enable a default world to be selected when launching the client"
+	name = "World",
+	description = "Enable a default world to be selected when launching the client and overlay your current world " +
+		"on the minimap globe",
+	tags = {"default", "world", "overlay"}
```

Comment:
this plugin is doing nothing when enabled, so make it disabled by default

Reply:
Forgot to add that line in my previous commit. It's there now. Thanks.